invenio_rdm_migrator/streams/communities/table_generator.py

```diff
@@ -26,7 +25,7 @@ def _generate_featured_community_id(data):
     return pid_pk()
 
 
-class CommunityTableGenerator(TableGenerator):
```

Comment:
(continuation on design questions...)

Each (or almost each) stream will need a csv table generator (for bulk/initial migration) and then another in memory one (which might be lighter and not yield every row/table. I'm considering if the table generator should have the `_generate_rows` as composition and then the streams define functions. This would make it more modular?